src/models/transactions/paymentChannelClaim.ts

```diff
@@ -0,0 +1,46 @@
+import { ValidationError } from "../../common/errors";
+import { BaseTransaction, GlobalFlags, verifyBaseTransaction } from "./common";
+
+export interface PaymentChannelClaimFlags extends GlobalFlags {
+    tfRenew?: boolean;
+    tfClose?: boolean;
+}
+
+export interface PaymentChannelClaim extends BaseTransaction {
+    TransactionType: "PaymentChannelClaim";
+    Flags?: number | PaymentChannelClaimFlags
```

Comment:
Does this overwrite the `| GlobalFlags` type option in BaseTransaction?

Reply:
Wondering the same thing - is this necessary given the `extends GlobalFlags`?

Reply:
Yeah, this should enforce that a PaymentChannelClaim has a PaymentChannelClaimFlags